com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/XR/XRSystem.cs

```diff
@@ -135,11 +135,13 @@ internal List<(Camera, XRPass)> SetupFrame(Camera[] cameras, bool singlePassAllo
                     // Disable vsync on the main display when rendering to a XR device
                     QualitySettings.vSyncCount = 0;
 
+#if ENABLE_VR && ENABLE_XR_MODULE
```

Comment:
hmm... Why do you need this, it doesn't compule? (cc @fabien-unity )

Reply:
I already fixed this here https://github.com/Unity-Technologies/Graphics/pull/1391/files  (please revert this bit as it will be redundant and it won't merge well automatically)